netmaster/objApi/apiController.go

```diff
@@ -569,12 +569,14 @@ func (ac *APIController) NetworkCreate(network *contivModel.Network) error {
 
 	// Build network config
 	networkCfg := intent.ConfigNetwork{
-		Name:       network.NetworkName,
-		NwType:     network.NwType,
-		PktTagType: network.Encap,
-		PktTag:     network.PktTag,
-		SubnetCIDR: network.Subnet,
-		Gateway:    network.Gateway,
+		Name:           network.NetworkName,
+		NwType:         network.NwType,
+		PktTagType:     network.Encap,
+		PktTag:         network.PktTag,
+		SubnetCIDR:     network.Subnet,
+		Gateway:        network.Gateway,
+		IPv6SubnetCIDR: network.Ipv6Subnet,
+		IPv6Gateway:    network.Ipv6Gateway,
```

Comment:
We add unit tests for contivModel objects  in netmaster/objApi/objapi_test.go
It might be good to add few unit tests for new IPv6 fields